environment/terraform.tfvars.json

```diff
@@ -34,7 +35,8 @@
       "symfony_env": "prod",
       "db_subnet_group": "private",
       "ec_subnet_group": "private",
-      "state_source": "preproduction"
+      "state_source": "preproduction",
+      "elasticache_count": 1
```

Comment:
It would be nice to also have two nodes in preproduction so we've got a production-like environment. Would that cost a huge amount financially?

Reply:
Ive made it 2. If we decide it's unneeded we can just change this value again.